app/src/main/AndroidManifest.xml

```diff
@@ -60,7 +60,8 @@
 
         <activity android:name=".splashScreen.SplashScreenActivity"
             android:theme="@style/SplashTheme"
-            android:screenOrientation="portrait">
+            android:screenOrientation="portrait"
+            android:exported="true">
```

Comment:
Are we expecting other apps to be able to launch ours or is there another reason this is set to true ?

Reply:
I think we don't expect that behavior. So, I can safely remove this `android:exported="true"` line?

Reply:
Yup, the exported value should be set to true only if we expect other applications to be able to launch ours. It is perfectly safe to remove it